core/src/cluster_info_vote_listener.rs

```diff
@@ -85,8 +85,8 @@ impl Service for ClusterInfoVoteListener {
 
 #[cfg(test)]
 mod tests {
-    use crate::locktower::MAX_RECENT_VOTES;
     use crate::packet;
+    use crate::tower_bft_consensus::MAX_RECENT_VOTES;
```

Comment:
I'm okay with calling this module `consensus.rs`. We could rename it to something more specific if we every had two consensus mechanisms living side by side, but I don't see that happening any time soon.